app/src/main/java/com/example/androidacademymsk/FragmentMoviesDetails.kt

```diff
@@ -20,5 +24,22 @@ class FragmentMoviesDetails : Fragment() {
         view.findViewById<ImageView>(R.id.backArrow).setOnClickListener {
             fragmentManager?.popBackStack()
         }
+        castRecycler = view.findViewById(R.id.castRecycler)
+        castRecycler?.layoutManager = LinearLayoutManager(context, RecyclerView.HORIZONTAL, false)
+        castRecycler?.setHasFixedSize(true)
+        val adapter = CastAdapter()
+        adapter.setHasStableIds(true)
+        castRecycler?.adapter = adapter
+    }
+
+    override fun onStart() {
+        super.onStart()
+        updateCastData()
+    }
+
+    private fun updateCastData() {
+        (castRecycler?.adapter as? CastAdapter)?.apply {
```

Comment:
you can just keep a reference to the adapter as a field in this class

Reply:
Done

Reply:
Done